### ISSUE TITLE
Failing to profile the nix ruby binary

### ISSUE THREAD

**Issue by viraptor** [rbspy/rbspy#374]
FTR, this is an FYI and doesn't block me. I may end up debugging it myself at some point.

Profiling the ruby binary from nixpkgs (2.7.6, 3.0.4, 3.1.2) fails with the standard "Failed to locate Ruby version symbol" error. It's a stripped binary, linking against local versions of every library. (not using distro files)

I don't have further info at this point, but may update in a while.

**Comment by omnibs** [comment]
@viraptor did you end up being able to use `rbspy` with a ruby binary from nixpkgs, or have any pointers?

I hit this today and found your issue

**Comment by viraptor** [comment]
No, I did something else. Haven't really dug into this. It's probably trivial to fix by skipping stripping, but I'm not sure that's something that should live in the repo either...

**Comment by laurentS** [comment]
This is an old issue, but I hit it this week, and with a bit of debugging `rbspy` figured out where the problem was. See PR above. It's not about skipping stripping, there was a bug in `rbspy`.

**Comment by acj** [comment]
Keeping this open until we can confirm the fix.

**Comment by acj** [comment]
The fix is in [v0.46.0](https://github.com/rbspy/rbspy/releases/tag/v0.46.0). Please give it a try.

**Comment by laurentS** [comment]
I've opened a PR in `nixpkgs` (https://github.com/NixOS/nixpkgs/pull/514070) to get the new release included there.
All tests I've done so far are good (things that didn't work with 0.45 work with 0.46) but hopefully someone else will test it out while reviewing the PR. I'd say we can close this issue when the PR is merged.
